src/components/interface/netshmem/lib.c

```diff
@@ -0,0 +1,75 @@
+#include "netshmem.h"
```

Comment:
Some documentation as to what this library actually does would be useful. I read through most of the code, and still don't really understand its purpose. It likely has a good one, I just don't get it ;-)